edx_proctoring/static/proctoring/js/views/proctored_exam_view.js

```diff
@@ -71,6 +76,22 @@ var edx = edx || {};
                 "credit eligibility in this course.\n");
```

Comment:
@chrisndodge @muhhshoaib I thought I had provided suggested text for lines 72-76 in another PR.